src/documentation/netex_idfm_to_ntfs.md

```diff
@@ -165,21 +165,26 @@ Be careful, `physical_mode_id` and `commercial_mode_id` are **not** prefixed.
 
 Here is the mapping table between a TransportMode value and the corresponding `physical_mode` and `commercial_mode`:
 
-TransportMode in Netex-IDFM | physical_mode_id | physical_mode_name | commercial_mode_id | commercial_mode_name 
---- | --- | --- | --- | ---
-air | Air | Avion | Air | Avion
-bus | Bus | Bus | Bus | Bus
-coach | Coach | Autocar | Coach | Autocar
-ferry | Ferry | Ferry | Ferry | Ferry
-metro | Metro | Métro | Metro | Métro
-rail | LocalTrain | Train régional / TER | LocalTrain | Train régional / TER
-trolleyBus | Tramway | Tramway | TrolleyBus | TrolleyBus
-tram | Tramway | Tramway | Tramway | Tramway
-water | Boat | Navette maritime / fluviale | Boat | Navette maritime / fluviale
-cableway | Tramway | Tramway | CableWay | CableWay
-funicular | Funicular | Funiculaire | Funicular | Funiculaire
-lift | Bus | Bus | Bus | Bus
-other | Bus | Bus | Bus | Bus
+TransportMode in Netex-IDFM | TransportSubMode in Netex-IDFM | physical_mode_id | physical_mode_name | commercial_mode_id | commercial_mode_name | Note
+--- | --- | --- | --- | --- | --- | ---
+air | _any_ | Air | Avion | Air | Avion |
+bus | _any_ | Bus | Bus | Bus | Bus |
+coach | _any_ | Coach | Autocar | Coach | Autocar |
+ferry | _any_ | Ferry | Ferry | Ferry | Ferry |
+metro | _any_ | Metro | Métro | Metro | Métro |
+rail | railShuttle | RailShuttle | Orlyval, CDG VAL | RailShuttle | Orlyval, CDG VAL |
+rail | suburbanRailway | LocalTrain | Train Transilien | LocalTrain | Train Transilien |
+rail | regionalRail | Train | TER Intercités | regionalTrain | TER |
+rail | interregionalRail | Train | TER Intercités | interregionalTrain | Intercités |
```

Comment:
```suggestion
rail | regionalRail | Train | TER / Intercités | regionalTrain | TER |
rail | interregionalRail | Train | TER / Intercités | interregionalTrain | Intercités |
```

Reply:
Done in f89552b55e25c336cbbbceebb98d74fc12388dd6.